integrations/system/network-interface-monitoring/README.md

```diff
@@ -0,0 +1,175 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The network interface monitoring integration provides metrics collection and alerting for network interface activity.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `network-interface-health` ([check])
+* `sensu/network-interface-check` ([asset])
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+This integration is compatible with the [Sumo Logic Host monitoring dashboard][sumo-host-dashboard-link] (included w/ [Sensu Plus][sensu-plus]).
+
+![](img/dashboard.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Add one of the following [subscriptions] to [agents] that should run this check.
```

Comment:
Do the README subscriptions match the CheckConfig?

Reply:
Oops, good catch.